### PR TITLE
DALLE image generator in Media Library

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/templates_and_media/media_library.md [hunk 1]
```diff
@@ -132,8 +125,27 @@ Android rich notifications do not support GIFs.
...
 
+## Accessing the Media Library from a message composer
+
+The Media Library acts as your dashboard's centralized location for assets, as all images are uploaded directly to it. This lets you re-use images across different messages.
+
+![Two common ways of accessing the Media Library depending on the message composer. One shows the email Drag and Drop Editor with the title "Images and GIFs" and a button to "Add from Media Library". The other shows the standard editors, such as push and in-app messages, with the title "Media" and a button to "Add Image".][1.5]{: style="border:none"}
+
+## Generate an image using AI {#generate-ai}
+
+You can generate images for your Media Library using [DALL·E 2](https://openai.com/dall-e-2/), an AI system from OpenAI that can create realistic images and art from a description in natural language. Each request generates four variations of your prompt, and you can generate images 10 times per day.
```
Comment (borisrev) [inline _docs/_user_guide/engagement_tools/templates_and_media/media_library.md:136]: "your company can generate images 10 times per day. This total applies to all users associated with your Braze instance." or something to that effect?

**Review by borisrev — CHANGES_REQUESTED** [review]
lgtm except for one comment/clarification! thanks Bre!

File: _docs/_user_guide/engagement_tools/templates_and_media/media_library.md (hunk 1 above)
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/templates_and_media/media_library.md:136]:
```suggestion
You can generate images for your Media Library using [DALL·E 2](https://openai.com/dall-e-2/), an AI system from OpenAI that can create realistic images and art from a description in natural language. Each request generates four variations of your prompt, and your company can generate images 10 times per day. This total applies to all users in your company.
```

Comment (bre-fitzgerald) [reply _docs/_user_guide/engagement_tools/templates_and_media/media_library.md]: Added!

**Review by KellieHawks — APPROVED** [review]

**Review by borisrev — APPROVED** [review]

**Pull request merged**